catalog/app/containers/Bucket/PackageCopyDialog.js

```diff
@@ -52,6 +56,45 @@ async function requestPackageCopy(
   }
 }
 
+const useCopyDataSwitcherStyles = M.makeStyles((t) => ({
+  root: {
+    position: 'absolute',
+    right: t.spacing(-2),
+    top: 0,
+  },
+}))
+
+function CopyDataSwitcher({ input }) {
+  const classes = useCopyDataSwitcherStyles()
+
+  const handleChange = React.useCallback(
+    (event) => input.onChange(event.target.checked),
+    [input],
+  )
+
+  return (
+    <M.FormControlLabel
+      className={classes.root}
+      control={
+        <M.Switch
+          color="primary"
+          size="small"
+          checked={input.value}
+          onChange={handleChange}
+        />
+      }
+      label="Full copy"
```

Comment:
probably "Copy data" would be better (@akarve wdyt?). also, it would be nice to have some hint describing the purpose of this switch in more detail (like a question mark with a popup hint maybe)

Reply:
I changed the text to `Files will be copied from "${sourceBucket}" to "${targetBucket}"` when copy-data is active, and `Files will persist in "${sourceBucket}", but links will be copied` when copy-data is inactive